test/unit/cog_api/http/groups_test.exs

```diff
@@ -30,6 +30,17 @@ defmodule CogApi.HTTP.GroupsTest do
           assert found_group.id == created_group.id
         end
       end
+
+      it "includes the users for the group" do
```

Comment:
I considered it but didn't think it was really necessary since we're storing them directly on the object, that happens for free. Do you think we should? I'm fine with adding one if you feel differently.